storybookjs/storybook
# Core: Add static dir path mappings

Issue: #714

## What I did

Added the static path mapping feature.

## How to test

- Is this testable with Jest or Chromatic screenshots? Unsure
- Does this need a new example in the kitchen sink apps? Maybe
- Does this need an update to the documentation? Should be in the commit itself but maybe not enough?

If your answer is yes to any of these, please make sure to include it in your PR.
